src/libraries/Microsoft.PowerFx.Connectors/Tabular/Capabilities/ColumnCapabilitiesBase.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.PowerFx.Connectors
     [JsonConverter(typeof(AbstractTypeConverter<ColumnCapabilitiesBase>))]
     internal abstract class ColumnCapabilitiesBase
     {
+        public abstract override int GetHashCode();        
```

Comment:
>public abstract override int GetHashCode();    [](http://example.com/codeflow?start=8&length=46)

we should not need this. 
This is just a poco - the comparison check could be done at a higher semantic level.